tasks/orca_catalog_reporting/orca_catalog_reporting.py

```diff
@@ -133,11 +133,11 @@ def get_catalog_sql() -> text:
                 (:granule_id is null or cumulus_granule_id=ANY(:granule_id)) and 
                 (:start_timestamp is null or (EXTRACT(EPOCH FROM date_trunc('milliseconds', cumulus_create_time) AT TIME ZONE 'UTC') * 1000)::bigint>=:start_timestamp) and 
                 (:end_timestamp is null or (EXTRACT(EPOCH FROM date_trunc('milliseconds', cumulus_create_time) AT TIME ZONE 'UTC') * 1000)::bigint<:end_timestamp)
-            ORDER BY cumulus_granule_id
             ) as granule_ids
             JOIN
                 granules ON granule_ids.cumulus_granule_id = granules.cumulus_granule_id
         ) as granules
+    ORDER BY cumulus_granule_id, provider_id
```

Comment:
Jenny L mentioned that it may be possible in the future to have identical granule ids in different providers. Can remove provider_id ordering if we don't want to deal with that right now.

Reply:
ok by me to keep it in.